specification/azurefleet/AzureFleet.Management/examples/2023-11-01-preview/Fleets_CreateOrUpdate_MaximumSet.json

```diff
@@ -765,7 +765,8 @@
     },
     "201": {
       "headers": {
-        "Azure-AsyncOperation": "https://contoso.com/operationstatus"
```

Comment:
Any reason you changed 2023-11-01 preview version of the API?

Reply:
I was missing, so updated the preview version also, as suggested.

Reply:
Examples can be changed.